checkbox-test-plans/units/gnome-clocks.pxu

```diff
@@ -0,0 +1,18 @@
+plugin: manual
+id: gnome-clocks
+flags: simple
+_description:
+ PURPOSE:
+       Check that gnome-clocks runs and works correctly
+ STEPS:
+      1. Run gnome-clocks from the desktop icon. Does the icon look correct?
+      2. Make sure hamburger menu -> Help brings up the Help page
+      3. Add a world clock and verify the time is correct
```

Comment:
Maybe it would be better to specify to add two specific cities and specify the expected time difference.

Reply:
good idea